docs/_pages/releases.md

```diff
@@ -12,6 +12,7 @@ sidebar:
 **What's New**
 * Added `WithParameterName` extension to ease asserting on the parameter name for a thrown `ArgumentException` - [#1466](https://github.com/fluentassertions/fluentassertions/pull/1466).
 * Added `NotCompleteWithinAsync` to `TaskCompletionSourceAssertions` - [#1474](https://github.com/fluentassertions/fluentassertions/pull/1474).
+* Major overhaul on how enums are handled, see the [Migration Guide](/upgradingtov6) for more details - [#1479](https://github.com/fluentassertions/fluentassertions/pull/1479).
```

Comment:
🤔 Isn't this a breaking change?

Reply:
It includes both new assertions and breaking changes, but I thought putting in both places was maybe too much. Do you think it fits better under breaking changes?

Reply:
Added the new assertions to "What's new" and the link to upgrade notes in "Breaking changes"